content/foundations/color.mdx

```diff
@@ -2,64 +2,187 @@
+| `border.done`            | Use to style text that refers to completed tasks or workflows.                |
+| `border.[ROLE].emphasis` | Use to highlight the most important item of a view or an interface            | 
+
+### Functional system in action
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+    <Box>
+    <img
+        width="464"
+        alt="image with combination of foundational variables"
+        src="https://user-images.githubusercontent.com/6951037/145223860-4db73d78-8791-44db-b94d-c7966ce13fab.png"
+    />
+  <Caption>A pairing of foundations is bg.subtle, border.default, and border.muted </Caption>
+    </Box>
+    <Box>   
+        <img
+            width="464"
+            alt="img-bg-inset"
+            src="https://user-images.githubusercontent.com/6951037/145223862-2ea9027c-1474-47d0-8caf-8ab8d93a841b.png"
+        />
+        <Caption>Thinking about elevation bg.inset would the underground. It's meant to create a feeling of focus or depth</Caption>
+    </Box>
+</Grid>
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+    <Box>
+    <img
+        width="464"
+        alt="img-onEmphasis"
+        src="https://user-images.githubusercontent.com/6951037/141304060-1e3d5285-f80f-4363-8c07-e5865c6fe073.png"
+    />
+  <Caption>fg.onEmphasis pairs with bg.[role].emphasis variables. In this example, fg.onEmphasis paired with bg.accent.emphasis </Caption>
+    </Box>
+    <Box>   
+        <img
+            width="464"
+            alt="img-color-roles-example"
+            src="https://user-images.githubusercontent.com/6951037/141304173-5f919074-b5ae-4c0b-b5d2-8a2cd6ea2d03.png"
+        />
+        <Caption>En example of an alert that pairs bg.success, fg.success, and border.success </Caption>
+    </Box>
+</Grid>
+
+### Combining Colors
+Not all colors pair well with each other. There are combinations of backgrounds and foregrounds that guarantee compliance with WCAG contrast guidelines and a wide range of hierarchical relationships between elements. 
+
+#### Pairing color roles
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+    <Box>
+    <img
+        width="464"
+        alt="Correct colored text and background combinations"
+        src="https://user-images.githubusercontent.com/6951037/141516234-0e267cba-a417-408c-958b-48fcfd1e76b6.png"
+    />
+  <Caption>Don’t combine color text with bg.subtle or bg.emphasis. Colored foregrounds should be combined with their matching backgrounds or bg.default.</Caption>
+    </Box>
+    <Box>   
+        <img
+            width="464"
+            alt="Incorrect colored text and background combinations"
+            src="https://user-images.githubusercontent.com/6951037/143043985-2fb49bb6-9d69-4cd8-9e26-ef5c81147dda.png"
+        />
+        <Caption>Use a link in a footer to provide access to a related page.</Caption>
+    </Box>
+</Grid>
```

Comment:
**Unrelated to your question:** The semantic `caption` element only applies to semantic tables. If there is a caption for an image it should be `<figcaption>` and it is nested within a `<figure>` element along with the `<img>` [Reference: 5.3.2.6 Naming Tables and Figures with Captions](https://www.w3.org/TR/wai-aria-practices-1.2/#naming_with_captions).

**Related to your question:** Text for an image will usually be preferred outside of an image so that it can be highlighted, copied, and to manipulate size for better reading. 

In this case, it's a summary of what one is seeing but it's not super helpful in describing the image on screen. If it's purely a visual aid, it should have `alt=""`, but if it helps, it should be described with a bit more detail. For example, just looking at the code here, I have a hard time picturing what's in the image other than one is right and one is wrong, and that's already described in the text. If it's helpful to note that the visual has a squares with white backgrounds and separate green, purple, and X text, then label it. If not, leave the label empty. 👍🏻 

Hope this helps! Happy to hop on a call and explain further.